### PR TITLE
Refs #29464 - hide the password value in CCC playbook

### PR DESCRIPTION
**Pull request by ares**
requires https://github.com/theforeman/foreman/pull/7559

### CODE REVIEW

**Review by adamruzicka — APPROVED** [review]
Works nicely, ACK

File: job_templates/configure_cloud_connector_-_ansible_default.erb [hunk 1]
```diff
@@ -7,11 +7,13 @@ template_inputs:
   input_type: user
   advanced: false
   value_type: plain
+  hidden_value: false
```
Comment (ezr-ondrej) [inline job_templates/configure_cloud_connector_-_ansible_default.erb:10]: this is a default, so it should not be necessary, right?

Comment (ares) [reply job_templates/configure_cloud_connector_-_ansible_default.erb:10]: it is not, however exporting the template would add it (same as e.g. advanced: false above)

**Comment by ezr-ondrej** [comment]
Thanks @ares and thanks @adamruzicka :+1:

**Pull request merged**